ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -30,6 +29,7 @@ import {
     timesOfDay,
     transformDurationLongForm,
 } from './integrationUtils';
+import { AuthMachineToMachineConfig } from '../../../services/MachineAccessService';
```

Comment:
nit: you can use the `services` directory as if it were a top-level directory, which makes those imports easier to read and maintain.

```
import { AuthMachineToMachineConfig } from 'services/MachineAccessService';
```